beta-pipelines/broad/somatic/single_sample/wgs/gdc_genome/GDCWholeGenomeSomaticSingleSample.wdl

```diff
@@ -568,9 +590,11 @@ task collect_insert_size_metrics {
   input {
     File input_bam
     String output_bam_prefix
+    Int additioinal_memory_mb = 0
+    Int additional_disk_gb = 0
   }
-
-  Int disk_size = ceil(size(input_bam, "GiB")) + 20
+  Int mem = 7500 + additioinal_memory_mb
```

Comment:
No dynamic sizing?

Reply:
this is the same size we have on our germline task, I wondered if it would be a waste to scale this with the size? Disk does scale. 
Happy to change it though